src/reporting.js

```diff
@@ -68,8 +68,9 @@ async function getReport(console, domain, mediaId, eventContentFile, opts) {
 }
 
 async function scannedDownload(req, res, domain, mediaId, eventContentFile, opts) {
-    opts.withCleanFile = function(filePath, fn) {
+    opts.withCleanFile = function(filePath, headers, fn) {
         req.console.info(`Sending ${filePath} to client`);
+        res.set(headers);
```

Comment:
You probably don't want to blindly pass all the headers though, eg. the `Connection` header. You probably want to whitelist some.